android_loomo_ros/src/main/java/org/loomo/ros/TFPublisher.java

```diff
@@ -216,6 +217,63 @@ private TransformStamped baseLinkToNeckTransform(Time time) {
         return transformStamped;
     }
 
+    // Added by Nilesh ******************************************************************************
+    private Quaternion setRPY(double roll, double pitch, double yaw)
```

Comment:
Full name such as `setRollPitchYaw` is preferred, but in this case I think it's ok